sources/chargen.js

```diff
@@ -872,20 +872,21 @@ $(document).ready(function() {
         const previewToDraw = {};
         const animation =  $(this).data(`layer_1_custom_animation`);
 
-        if($(this).data(`layer_1_${getBodyTypeName()}`) === undefined) {
-          let imageLink = $(this).data(`layer_1_${getBodyTypeName()}`);
+        const bodyTypeName = getBodyTypeName();
+        if ($(this).data(`layer_1_${bodyTypeName}`) === undefined) {
+          let imageLink = $(this).data(`layer_1_${bodyTypeName}`);
```

Comment:
This logic here. Am I overseeing sth or is this wrong?

It states:

```
if ($(this).data(`layer_1_${bodyTypeName}`) === undefined) {
          let imageLink = $(this).data(`layer_1_${bodyTypeName}`);
```
So that will mean that imageLink will always be undefined.. Do you get that?

Reply:
Nope you're right. I missed that.

Reply:
See #90 first commit.